tests/conftest.py

```diff
@@ -156,6 +171,9 @@ def test_mytest(app, db, es):
         if 'endpoint' in request.param:
             app.config['RECORDS_REST_ENDPOINTS']['recid'].update(
                 request.param['endpoint'])
+        elif 'records_rest_endpoints' in request.param:
```

Comment:
Kind of, because `records_rest_endpoints` somehow overrides the weaker `endpoint`. But OK.